pages/tidal tools/sourcecode.md

```diff
@@ -1,120 +1,107 @@
    ```
 
-   Note:
+   The `--offline` flag indicates that the results are outputted to a local file, instead of being uploaded.
+
+   After the analysis is completed, you will find the results in a file called `code-analysis-<DATE>-<TYPE>.json`.
 
-   - The `--offline` flag indicates to Tidal Tools that the output needs to be stored in a file instead of being uploaded.
 
-   After the analysis is completed, you will find an artifact file called `code-analysis-<DATE>-<TYPE>.json` that can then be transferred into a machine with internet connectivity.
+5. Transfer the results file from the analysis to the machine with internet access.
 
-5. Back to the machine with internet access, you can now upload your results to your workspace with this command.
+5. On the machine with internet access, upload the results to your workspace:
 
 
    ```bash
    tidal analyze code upload [file_name] --app-id [app_id_for_your_application]`
    ```
 
-   You should receive confirmation that the upload has been completed and can navigate to Tidal to see the results.
+   You should receive confirmation that the upload has been completed and can navigate to Tidal Accelerator to see the results.
 
-{% include tip.html content="Need to run code analysis on a entire set of applications all from one machine? Run [this command and easily create a directory for every application already in Tidal Accelerator](https://github.com/tidalmigrations/gists/blob/master/make_source_code_dirs.sh)" %}
+{% include tip.html content="Need to run code analysis on an entire set of applications all from one machine? Run [this command and easily create a directory for every application already in your Tidal Accelerator workspace](https://github.com/tidalmigrations/gists/blob/master/make_source_code_dirs.sh)." %}
 
 ### Why Docker?
 
-You need to install Docker in order to complete the source code analysis. This is because the analysis uses several system dependent software libraries. 
+You need to install Docker to complete the source code analysis. This is because the analysis uses several system-dependent software libraries. 
 By using Docker, the analysis can use those libraries without requiring you to install any other dependencies.
 
 ### What about security?
 
-The entire analysis takes place _locally on your machine_. The **only** data that is captured and sent from the analysis are the results of the analysis and metadata. **No source code, files or the contents of any files on your machine are ever copied or sent anywhere.**
+**The entire code analysis process takes place locally on your machine.** It will scan your files locally looking for common patterns and information when modernizing. The only data that is captured and sent from the analysis are the results of the analysis and the associated metadata. You can view all of the raw analysis results that are sent to Tidal Accelerator before they are sent, by viewing the contents of the results file locally.
```

Comment:
I wanted to add, reiterate(?), the way to get local results is through offline analysis. Any other changes made were due to Grammarly 🤷 
```suggestion
**The entire code analysis process takes place locally on your machine.** It will scan your files locally looking for common patterns and information when modernizing. The only data that is captured and sent from the analysis are the results of the analysis and the associated metadata. You can view all raw analysis results sent to Tidal Accelerator before they are sent by performing an offline analysis and viewing the results file locally.
```

Reply:
thanks for that, good improvement. Ironically, the `of the` was _added by_ Grammarly when I did a final review of it! haha. I agree, with it on this pass, it's simpler. 

I also updated the security section on the overview guide, as it repeats identically there too.